components/admin_console/admin_console.test.tsx

```diff
@@ -33,6 +33,8 @@ describe('components/AdminConsole', () => {
         },
         showNavigationPrompt: false,
         isCurrentUserSystemAdmin: false,
+        currentUserHasAnAdminRole: false,
+        globalstate: {},
```

Comment:
should this be `globalState`?

Reply:
This actually needs to be removed, it's from something I removed from the original PR.

Reply:
Removed. Thanks.